src/FakeItEasy/Creation/CastleDynamicProxy/CastleDynamicProxyInterceptionValidator.cs

```diff
@@ -34,7 +34,7 @@ private static string GetReasonForWhyMethodCanNotBeIntercepted(MethodInfo method
                 var explicitImplementation = method.Name.Contains('.');
                 if (!explicitImplementation)
                 {
-                    return "Sealed methods can not be intercepted.";
+                    return "Method should be marked as virtual, abstract or part of an Interface in oreder to be intercepted.";

```

Comment:
I tend to agree with you, @thomaslevesque. Looking at how the message is presented in context:

```
FakeItEasy.Specs.ConfigurationSpecs.CallToSetSealedOnFake() [02] When I start to configure a sealed property setter on the fake [FAIL]
   FakeItEasy.Configuration.FakeConfigurationException :                                                                              
                                                                                                                                      
     The current proxy generator can not intercept the specified method for the following reason:                                     
       - Method should be marked as virtual, abstract or part of an Interface in oreder to be intercepted.                            ```
```

"Method should be marked…" doesn't really flow after the intro. Sorry, @chrismatheson, I think @thomaslevesque's version initial sentence works better in this case. 

I think your second sentence is quite good (except for the missing serial comma :wink:).
I'm a little concerned that we're going to end up turning the error messages into a .NET tutorial, but maybe that's not a bad thing.

@chrismatheson, as the person who was actually negatively affected by the original message, do you think "Non-virtual methods can not be intercepted. Virtual methods are methods explicitly marked virtual, overriding methods, abstract methods**,** and interface methods."
would've been more helpful? Helpful enough?